app/main.py

```diff
@@ -1,8 +1,49 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int, clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, car_list: list[Car]) -> float:
+        income = 0
+
+        for auto in car_list:
+            if auto.clean_mark < self.clean_power:
+                income += self.wash_single_car(auto)
+
+        return round(income, 1)
+
+    def calculate_washing_price(self, auto: Car) -> float:
+        return round(
+            auto.comfort_class * (self.clean_power - auto.clean_mark)
+            * self.average_rating / self.distance_from_city_center , 2)
```

Comment:
In the `calculate_washing_price` method, there is a potential division by zero error if `distance_from_city_center` is zero. Consider adding a check to handle this case to prevent runtime errors.